spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -239,4 +240,124 @@
       end
     end
   end
+
+  describe '#call' do
```

Comment:
What do you think about combining this with the other `#call` describe block in this spec? Or changing this line so that there aren't two describe blocks with the same name in one file?